docs/api/autogluon.task.rst

```diff
@@ -51,18 +53,17 @@ Prediction tasks built into AutoGluon such that a single call to `fit()` can pro
    TextPrediction
 
 
-:hidden:`TabularPrediction`
+:hidden:`TabularPredictor`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .. automodule:: autogluon.tabular
 
-.. autoclass:: TabularPrediction
-   :members: fit, load, Predictor, Dataset
-   :no-inherited-members:
-   :exclude-members: run_fit
+.. autoclass:: TabularPredictor
+   :members:
+   :inherited-members:
```

Comment:
current docs show many attributes of TabularPredictor (eg. class_labels, eval_metric, etc.) without any explanation of these attributes.

Reply:
Will address in follow-up